bitbots_blackboard/src/bitbots_blackboard/capsules/world_model_capsule.py

```diff
@@ -390,3 +399,111 @@ def get_distance_to_xy(self, x, y):
         u, v = self.get_uv_from_xy(x, y)
         dist = math.sqrt(u ** 2 + v ** 2)
         return dist
+
+    ############
+    # Obstacle #
+    ############
+
+    def local_costmap_callback(self, msg):
+        self.local_obstacle_map = msg
+
+    def obstacle_value_at_relative_xy(self, x, y, area=1):
```

Comment:
area is currently not used. I would remove it if it is not necessary as it is the reason the lambda is needed when choosing the kick direction